Tasks/XamarinAndroid/task.json

```diff
@@ -12,25 +12,24 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 8
+        "Patch": 10
```

Comment:
Multiple updates to version number were due to multiple changes made to the task in multiple commits, I guess since now they are all combined into one release, it might be ok to just up by one.